Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
-  end
-
   def caveats; <<-EOS.undent
-    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni.
+    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni/decks/.
     EOS
   end
 
+  def plist; <<-EOS.undent
+   <?xml version="1.0" encoding="UTF-8"?>
+   <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+   <plist version="1.0">
+   <dict>
+     <key>Label</key>
+       <string>#{plist_name}</string>
+     <key>Program</key>
+       <string>#{opt_bin}/ooniprobe</string>
+     <key>ProgramArguments</key>
+     <array>
+       <string>-i</string>
+       <string>#{HOMEBREW_PREFIX}/share/ooni/decks/current.deck</string>
```

Comment:
`opt_prefix` will always == `HOMEBREW_PREFIX/Cellar/formula/current_linked_ver` and `opt_share` will always == `HOMEBREW_PREFIX/Cellar/formula/current_linked_ver/share`, essentially.

The opt helpers are just a way to reference a path that doesn't break every revision/update.